tests/test_icon.py

```diff
@@ -0,0 +1,21 @@
+from django.test import TestCase
+from django.utils import translation
+from django.utils.translation import gettext_lazy as _
+
+from django_icons import icon
+from django_icons.renderers import BaseRenderer
+
+
+class IconTest(TestCase):
+    """Test the icon function."""
+
+    def test_laziness(self):
+        self.assertEqual(
+            icon("user", title=_("user"), renderer=BaseRenderer),
+            '<i class="user" title="user"></i>',
+        )
+        with translation.override("nl"):
+            self.assertEqual(
+                icon("user", title=_("user"), renderer=BaseRenderer),
+                '<i class="user" title="gebruiker"></i>',
+            )
```

Comment:
This test does not proof lazyness, I'd expect something like this to work:

```suggestion
        user_icon = icon("user", title=_("user"), renderer=BaseRenderer)
        with translation.override("nl"):
            self.assertEqual(user_icon, '<i class="user" title="gebruiker"></i>')
```